frontend/src/app/modules/root/root-reducer.js

```diff
@@ -18,7 +18,10 @@ const initialState = {
 
 const usersLoaded = (state, { users }) => ({
   ...state,
-  users,
+  users: users.map((user) => {
+    user.id = Number(user.id)
```

Comment:
in general, mutating maps are a big no-no. with this map, you're editing the original user object.
```

users.map (user => {
  return {...user, id: Number(user.id)}
})
```

would be a way to go

Reply:
actually - yesterday we agreed not to fix it on backend due to the hanging PR. I was wrong. Consistent and good design is more important than efficiency. Could rather you fix it on the backend and ignoring the conflict in the hanging PRs? The authors of the hanging PRs should do the rebase